src/components/AttractionCard/AttractionCard.js

```diff
@@ -1,8 +1,25 @@
 import styled from 'styled-components/macro'
+import { ReactComponent as LikeSVG } from '../../icons/liked.svg'
+import { ReactComponent as UnlikeSVG } from '../../icons/unliked.svg'
 
-export default function AttractionCard({ name, image }) {
+const like = <LikeSVG />
+const unlike = <UnlikeSVG />
+
+export default function AttractionCard({
+  name,
+  image,
+  likePlace,
+  likedPlaces,
+}) {
   return (
     <Card key={name}>
+      <LikeButton
+        role="button"
+        aria-label="toggle-like"
+        onClick={() => likePlace(name)}
+      >
+        {likedPlaces.includes(name) ? like : unlike}
+      </LikeButton>
       <AttractionName>{name}</AttractionName>
       <Image src={image} width="335" alt=""></Image>
```

Comment:
You should set `height` as well here.
```suggestion
      <Image src={image} width="335" height="335" alt="" />
```

Reply:
👍🏻